run-videochat.sh

```diff
@@ -346,13 +346,23 @@ if ! iw_server_is_started; then
     error "$MESSAGE\nPlease install and open IP Webcam in your phone and start the server.\nMake sure that values of variables IP, PORT, CAPTURE_STREAM in this script are equal with settings in IP Webcam."
 fi
 
+# Test if audio server is running on pipewire
+if pactl info | grep -q PipeWire; then
+    if [ "$CAPTURE_STREAM" = v ]; then
+        :
+    else
+    # currently setting audio sinks errors out, so give user a workaround
+        error "Only video streams on Pipewire are currently supported. Audio is a WIP. Please set CAPTURE_STREAM value to v."
+    fi
+fi
+
 if [ $CAPTURE_STREAM = a -o $CAPTURE_STREAM = av ]; then
     # idea: check if default-source is correct. If two copy of script are running,
     # then after ending first before second you will be set up with $SINK_NAME.monitor,
     # but not with your original defauld source.
     # The same issue if script was not end correctly, and you restart it.
-    DEFAULT_SINK=$(pacmd dump | grep set-default-sink | cut -f2 -d " ")
-    DEFAULT_SOURCE=$(pacmd dump | grep set-default-source | cut -f2 -d " ")
+    DEFAULT_SINK=$(pactl info | grep "Sink" | cut -f3 -d " ")
```

Comment:
Likewise, this would also have to be reworked to have a switch around the two systems.